### ISSUE TITLE
Optimisation issue

### ISSUE THREAD

**Issue by manishjon6** [zeelog/OTA#76]
Hii 
kindly look the optimisation issue in 31 aug patch, old 9 aug OTA more optimized but new one is less.
handset is lagging pls release optimised OTA UPDATE.

**Comment by dyno01** [comment]
I also agree . sir please fix this , while streaming on YouTube feel laggy and jitter and in UI also.

**Comment by ODRAUDESCORP** [comment]
Good morning, first of all, thank you very much for going ahead with the mido updates, the rom is almost perfect but at times you feel slow transactions in applications, there are also problems with the 4k video recording, the recordings are slow
